applications/asset_tracker/src/env_sensors/env_sensors.c

```diff
@@ -214,8 +214,8 @@ void env_sensors_set_send_interval(const uint32_t interval_s)
 	if (data_send_interval_s) {
 		/* restart work for new interval to take effect */
 		env_sensors_poll();
-	} else if (k_delayed_work_remaining_get(&env_sensors_poller) > 0) {
-		k_delayed_work_cancel(&env_sensors_poller);
+	} else if (k_ticks_to_ms_ceil32(k_work_delayable_remaining_get(&env_sensors_poller)) > 0) {
+		k_work_cancel_delayable(&env_sensors_poller);
```

Comment:
@rlubos In these situations. Should we just throw away the "reamaning_get" and cancel anyway?

Reply:
FWIW, I think you should. A safer version of this check already exists inside the cancel function.

Reply:
I agree with @trond-snekvik

Reply:
done